### PR TITLE
cmx token metadata & logos

### PR DESCRIPTION
**Pull request by trentdavis78**
## How to add token?

- [ ] Add metadata to [`tokens.json`](tokens.json)

```json
{
    "name": "<NAME>",
    "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "symbol": "<SYMBOL>",
    "account": "<CONTRACT NAME>",
    "chain": "eos"
}
```

- [ ] Add token logo `*.png` to [`./logos`](./logos) folder

### CODE REVIEW

**Comment by jafri** [comment]
Merge conflicts

**Comment by trentdavis78** [comment]
> 

Fixed. Sorry about that.

**Comment by jafri** [comment]
The tokens.json diff is too large, please only modify to add your token

**Comment by trentdavis78** [comment]
> The tokens.json diff is too large, please only modify to add your token

I dont know what you mean, can you rephrase that?

**Comment by trentdavis78** [comment]
Wait, I think I know what you mean. Stand by.

**Comment by trentdavis78** [comment]
> The tokens.json diff is too large, please only modify to add your token

This should be fixed now.
